app/controllers/reports/regions_controller.rb

```diff
@@ -46,10 +46,10 @@ def show
                                                         period: @period).call
       }
     else
-      @show_current_period = true
+      @show_current_period = false
```

Comment:
Is this used in the view? Can we rip it out and hard-code falses where necessary, since it can't be anything else?

Reply:
I don't wanna change too much while all of this is in flight (and we have views shared by old reports and new).

I think long term we need to pass in the `Period` object to this tables to really do things right, but that will be easier once the old stuff is all gone.